### PR TITLE
add Code of Conduct

### LINKED ISSUE #1209: Code of Conduct
I wonder why Phoenix doesn't have a Code of Conduct yet.
I'd suggest adding [the one from the Contributor Covenant](http://contributor-covenant.org/version/1/2/0/) as `CODE_OF_CONDUCT.md` and link to it from the [`README.md`](https://github.com/phoenixframework/phoenix/blob/master/README.md) and  [`CONTRIBUTING.md`](https://github.com/phoenixframework/phoenix/blob/master/CONTRIBUTING.md).

Any thoughts on that?

### PR DESCRIPTION
**Pull request by optikfluffel**
from http://contributor-covenant.org
to close #1209

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 Please take a moment to review this document in order to make the contribution
 process easy and effective for everyone involved!
+Also make sure you follow our [CODE_OF_CONDUCT.md](CODE_OF_CONDUCT.md), so everyone involved in this project feels safe and welcomed.
```
Comment (josevalim) [inline CONTRIBUTING.md]: Can you please copy the rewording I applied to Elixir?

File: README.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ API documentation is available at [http://hexdocs.pm/phoenix](http://hexdocs.pm/
 
 ## Contributing
 
-We appreciate any contribution to Phoenix, so check out our [CONTRIBUTING.md](CONTRIBUTING.md) guide for more information. We usually keep a list of features and bugs [in the issue tracker][1].
+We appreciate any contribution to Phoenix, that's following our [CODE_OF_CONDUCT.md](CODE_OF_CONDUCT.md), so check out our [CONTRIBUTING.md](CONTRIBUTING.md) guide for more information. We usually keep a list of features and bugs [in the issue tracker][1].
```
Comment (josevalim) [inline README.md]: The same here. Please copy the rewording here.

Comment (josevalim) [inline README.md]: I just changed it again due to more feedback so please make you grab the latest. :)

**Comment by optikfluffel** [comment]
Done :white_check_mark: + fixed one link to the issue tracker (in `README.md`) that was actually pointing to the slackin thingy.

**Comment by josevalim** [comment]
Ok, please squash and @chrismccord should check and merge!

**Comment by chrismccord** [comment]
:heart: :purple_heart: :yellow_heart:

**Pull request merged**